solutions/Making-choices-choices-with-functions-R.txt

```diff
@@ -0,0 +1,3 @@
+[1] "CM - 0.866"
```

Comment:
I agree that this is a shortcoming of the lesson. The challenge is to decide under what scenario this programming problem might arise. The two options to me are summary of data as data context or a report of data context. Arguably, this solution fits neither or those scenarios. I propose two alternates:

1) A vector with three values: `c(species, rsquared, significance)`

2) A statement that contains a variable: "Acacia with `species` has a branch ~ canopy relationship that is `significance`, `rsquared`".